src/Servers/Kestrel/perf/Microbenchmarks/Http2/Http2ConnectionBenchmarkBase.cs

```diff
@@ -56,6 +53,16 @@ public virtual void GlobalSetup()
         _httpRequestHeaders[HeaderNames.Scheme] = new StringValues("http");
         _httpRequestHeaders[HeaderNames.Authority] = new StringValues("localhost:80");
 
+        if (NumCookies > 0)
+        {
+            var cookies = new StringValues("0=1");
+            for (var index = 1; index < NumCookies; index++)
+            {
+                cookies = (StringValues)cookies.Append($"{index}={index + 1}");
```

Comment:
`StringValues`  doesn't have a conversion from `IEnumerable<string>`. It does have an implicit conversion from `string[]`, so the following should work (but I don't recommend it):

```suggestion
                cookies = cookies.Append($"{index}={index + 1}").ToArray();
```

`StringValues` also has a constructor taking `string[]`, I'd just build an array of the correct length initially and use that even though this part of the code isn't perf sensitive.